src/Razor/src/Microsoft.VisualStudio.RazorExtension/Snippets/SnippetService.cs

```diff
@@ -190,6 +201,25 @@ private ImmutableArray<SnippetInfo> ExtractSnippetInfo(SnippetLanguage language,
         }
     }
 
+    private static ImmutableHashSet<string> GetIgnoredSnippets(SnippetLanguage language, SnippetSetting snippetSetting)
+    {
+        if (language == SnippetLanguage.CSharp)
+        {
+            // In call cases for C# we want to filter out the built in snippets. The C#
+            // language server will handle returning these and isn't required for the
+            // Razor code to add them.
+            return s_builtInSnippets[s_CSharpLanguageId];
+        }
+
+        if (snippetSetting == SnippetSetting.All)
+        {
+            return ImmutableHashSet<string>.Empty;
+        }
+
+        Debug.Assert(language == SnippetLanguage.Html);
```

Comment:
Is `SnippetLanguage.Razor` and `SnippetSetting.Custom` a valid combo here?

Reply:
currently no, because we don't have "razor" snippets. I could remove that from the `SnippetLanguage` but I figured it just helped with completeness.

Reply:
Got it, that makes sense. Maybe a comment here would be helpful to indicate that Razor snippets don't exist yet, otherwise on first glance it looks like we missed a case

Reply:
You're telling me my code isn't self documenting??? 😲

(thanks, will add )